components/compliance-service/reporting/relaxting/reports.go

```diff
@@ -530,6 +499,87 @@ func (backend *ES2Backend) GetReport(esIndex string, reportId string,
 	return report, errorutils.ProcessNotFound(nil, reportId)
 }
 
+func convertControl(profileControlsMap map[string]*reportingapi.Control, reportControlMin ESInSpecReportControl, filters map[string][]string) *reportingapi.Control {
```

Comment:
i'm kinda tempted to also break the profile conversion code out of that big function so i can unit test it, but i'd rather not bloat this change with that. thinking i'll open an issue for it as cleanup/tech debt